blond/gpu/gpu_impedance.py

```diff
@@ -50,6 +50,10 @@ def gpu_induced_voltage_sum(self):
         for induced_voltage_object in self.induced_voltage_list:
                 induced_voltage_object.induced_voltage_generation(
                         beam_spectrum_dict)
+                try:
+                        induced_voltage_object.dev_induced_voltage
+                except:
```

Comment:
Very unsafe to have a try/except without specifying the exception.  It's best to specify what exception to catch or you risk hiding unexpected errors.